landing.ftd

```diff
@@ -0,0 +1,1027 @@
+-- import: fastn-community.github.io/set-2-ds as ds
+
+-- my-ds.page:
+full-width: true
+sidebar: false
+
+-- hero-with-image: We transform ideas into digital outcomes.
+cta-primary: Demo
+cta-primary-link: /
+cta-secondary: How it works
```

Comment:
@MeenuKumari28 add mouse-enter and mouse-leave to add hover effect on all buttons.